ml-algorithms/src/main/java/org/opensearch/ml/engine/function_calling/FunctionCallingFactory.java

```diff
@@ -27,7 +27,15 @@ public static FunctionCalling create(String llmInterface) {
             case LLM_INTERFACE_BEDROCK_CONVERSE_DEEPSEEK_R1:
                 return new BedrockConverseDeepseekR1FunctionCalling();
             default:
-                throw new IllegalArgumentException(String.format("Invalid _llm_interface: %s", llmInterface));
+                throw new IllegalArgumentException(
+                    String
+                        .format(
+                            "Invalid _llm_interface. Supported values are %s,%s,%s",
+                            LLM_INTERFACE_BEDROCK_CONVERSE_CLAUDE,
```

Comment:
From previous security reviews, suggest not contains these valid values in error message.